### PR TITLE
Anna Cho Day 8 Homework (without mimsind1.py)

### PR DESCRIPTION
**Pull request by feladie**

### CODE REVIEW

File: HW08_ch11_ex02d.py [hunk 1]
```diff
@@ -28,16 +28,54 @@ def invert_dict_old(d):
 
 
 def invert_dict_new(d):
-    pass
+    inverse = dict()
+    for key, value in d.items():
```
Comment (OhRobin) [inline HW08_ch11_ex02d.py]: cool. 
d.items returns a tuple

File: HW08_ch11_ex02b.py [hunk 2]
```diff
@@ -22,12 +22,41 @@ def print_hist_old(h):
 
 
 def print_hist_new(h):
-    pass
+    keys = []
+    for c in h.keys():
```
Comment (OhRobin) [inline HW08_ch11_ex02b.py]: cool! h.keys() and h will return the same thing

Comment (danielsgriffin) [inline HW08_ch11_ex02b.py]: @OhRobin
Sorta,
see this:
```
>>> h = {"key1": 1, "key2": 2}
>>> h  # the dictionary itself
{'key1': 1, 'key2': 2}
>>> h.keys()
dict_keys(['key1', 'key2'])  # iterable object
```
In a for loop they are treated the same, but they aren't.

File: mimsmind0.py [hunk 3]
```diff
@@ -0,0 +1,88 @@
+import random
```
Comment (danielsgriffin) [inline mimsmind0.py]: The instructions also ask for you to be able to accept a command-line argument for length.

**Comment by danielsgriffin** [comment]
Noting that this isn't complete, I'm closing to move this out of the queue. Please re-submit with mimsmind when ready. You can find it in closed pull requests. 

Good job on mimsmind0 - just need to be prepared to accept a command-line argument.

**Pull request closed without merging**